hazelcast/src/main/java/com/hazelcast/core/Member.java

```diff
@@ -258,4 +260,15 @@
      */
     void removeAttribute(String key);
 
+    /**
+     * Returns the Hazelcast codebase version of this member; this may or may not be different from the version reported by
+     * {@link ClusterService#getClusterVersion()}, for example when a node with a different codebase version is added to an
```

Comment:
Ah, now I get it. So the Member.version is the version of Member X on stored on another member.